src/app/trips/trip-thumbnail/trip-thumbnail.component.html

```diff
@@ -16,15 +16,15 @@ <h2 class="thumbnail-title font700 font-adjust">{{trip?.title}}</h2>
           </span>
 
           <div class="thumbnail_location">
-            Destination<span *ngIf="multipleLocations(trip?.locations)">s</span>:
+            Destination<span *ngIf="tripService.multipleLocations(trip.locations)">s</span>:
```

Comment:
angular performs different if you have change detection trigger on methods vs properties. It can be pretty slow if you have services being used for change detection. You should change this to have a property in the component.ts check just once on view init and set that value to a property.